components/data-apps/lock-streamlit-version/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: How to Lock Package Versions in Streamlit Applications in Keboola (Deploying from Git Repository)
+permalink: /components/data-apps/lock-streamlit-version/
+---
+
+* TOC
+{:toc}
+
+This guide explains how to lock package versions in Streamlit applications deployed from a GIT repository in the Keboola platform. 
+
+Locking package versions is essential for:   
```

Comment:
@hhanova can you help me to implement those links here please?
https://packaging.python.org/en/latest/tutorials/packaging-projects/ and https://docs.streamlit.io/knowledge-base/dependencies
Thank you.